spinalcordtoolbox/deepseg/models.py

```diff
@@ -100,7 +108,10 @@
          'models': ['findcord_tumor', 'model_seg_sctumor-edema-cavity_t2-t1_unet3d-multichannel']},
     'seg_exvivo_gm-wm_t2':
         {'description': 'Grey/white matter seg on exvivo human T2w.',
-         'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']}
+         'models': ['model_seg_exvivo_gm-wm_t2_unet2d-multichannel-softseg']},
+     'seg_gm_sc_7t_t2star':
+        {'description': 'SC/GM seg on T2*-weighted at 7T',
+         'models': ['model_7T_multiclass_gm_sc_unet2d']} 
```

Comment:
```suggestion
         'models': ['model_7t_multiclass_gm_sc_unet2d']} 
```

I believe this one also needs to be changed.